src/node.cc

```diff
@@ -3786,6 +3790,16 @@ static void ParseArgs(int* argc,
     index += args_consumed;
   }
 
+#if HAVE_OPENSSL
```

Comment:
I'm wondering if we shouldn't just error out immediately if `--use-openssl-ca` is passed and `HAVE_OPENSSL` is off.
/cc @nodejs/ctc